### PR TITLE
en, zh: update description for the alpha features

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 1]
```diff
@@ -3,10 +3,6 @@ title: Deploy a TiDB Cluster across Multiple Kubernetes Clusters
 summary: Learn how to deploy a TiDB cluster across multiple Kubernetes clusters.
 ---
 
-> **Warning:**
```
Comment (ran-huang) [inline en/deploy-tidb-cluster-across-multiple-kubernetes.md:6]: 这个部署方式是在 1.3 GA 的吗？在 1.2 和 1.1 的分支上请保留警告。

Comment (DanielZhangQD) [reply en/deploy-tidb-cluster-across-multiple-kubernetes.md:6]: 嗯嗯，CP 的时候会改一下

File: en/TOC.md [hunk 2]
```diff
@@ -45,7 +45,6 @@
     - [Run Containers as a Non-root User](containers-run-as-non-root-user.md)
   - Scale
     - [Manually Scale](scale-a-tidb-cluster.md)
```
Comment (ran-huang) [inline en/TOC.md:47]:
```suggestion
  - [Scale](scale-a-tidb-cluster.md)
```

**Review by KanShiori — APPROVED** [review]

File: zh/TOC.md [hunk 3]
```diff
@@ -45,7 +45,6 @@
     - [以非 root 用户运行](containers-run-as-non-root-user.md)
   - 扩缩容
     - [手动扩缩容](scale-a-tidb-cluster.md)
```
Comment (DanielZhangQD) [inline zh/TOC.md:47]:
```suggestion
    - [扩缩容](scale-a-tidb-cluster.md)
```

**Review by qiancai — APPROVED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**